### PR TITLE
Test

### PR DESCRIPTION
**Pull request by Niharika29**
This PR mainly adds tests/ApiHelperTest.php. The extra stuff you see is either for Travis or because I included the vendor directory so that travis can find everything it needs to work on the tests. 
For PHP 5.4, travis throws a strange error saying it cannot find a particular function. So I am not testing against it anymore. The tests all pass happily: https://travis-ci.org/wikimedia/popularpages

### CODE REVIEW

**Comment by Niharika29** [comment]
Removed the vendor directory and included the composer install in travis (I was being silly). Thanks Sam!

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
-/vendor
```
Comment (samwilson) [inline .gitignore]: Can be left in now.

File: composer.json [hunk 2]
```diff
@@ -4,12 +4,13 @@
 	"license": "MIT",
 	"homepage": "https://wikitech.wikimedia.org/wiki/Tool:Popular_Pages",
 	"require": {
-		"php" : ">=5.5.9",
+		"php" : "~5.6",
```
Comment (samwilson) [inline composer.json]: But labs is 5.5.9 still isn't it?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+language: php
+php:
+#  - '5.4' -- breaks for some weird reason (https://travis-ci.org/wikimedia/popularpages/jobs/216541216)
```
Comment (samwilson) [inline .travis.yml]: Is this a phpunit incompatibility? You might have to downgrade phpunit to ^4.8 (the last version for php >=5.3.3).

Comment (Niharika29) [inline .travis.yml]: I changed it. It's still giving an error without any helpful logs. https://travis-ci.org/wikimedia/popularpages/jobs/218174186

File: .gitignore (hunk 1 above)
Comment (kaldari) [inline .gitignore]: It's been added back in now (further down)

**Pull request merged**